go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/elastic/package-storage
 go 1.12
 
 require (
-	github.com/elastic/package-registry v0.11.0
+	github.com/elastic/package-registry v0.13.0
```

Comment:
Looks like the CI registry is a couple versions back. Is it ok to bump that too? I think this brings in the `data_stream` changes.

Reply:
Yes, I can't think of any reason not to. I don't think it was deliberate that the version here was out of sync with the version in the Dockerfile. 

@mtojek can you think of any reason the version here and the version in the Dockerfile should not be in sync always?

Reply:
++ on bumping this. It would be nice to only have 1 place to update it to not get them out of sync. Maybe elastic-package is going to solve this all for us ;-)